How_the_toolkit_works.md

```diff
@@ -4,11 +4,11 @@ How the toolkit works
 <p align="center"><img src="https://raw.githubusercontent.com/astro4dev/OAD-Data-Science-Toolkit/master/img/DST_logo_250px.png" alt="DST Logo"/></p>
 
 ## Step 1 - Adding toolkit content
-The toolkit content is stored on the github servers under the <a href="https://github.com/astro4dev/OAD-Data-Science-Toolkit" target="_blank">OAD-Data-Science-Toolkit</a> repository. Content is added by individual contributors by cloning the repository, making the changes, then creating a pull request. A description of the workflow can be found <a href="https://github.com/astro4dev/OAD-Data-Science-Toolkit/blob/master/How_to_upload_content.md" target="_blank">here</a>. The astro4dev data science working group are automatically informed about the new pull request. The request is then verified before it is added to the main astro4dev repository.
+The toolkit content is stored on the github servers under the <a href="https://github.com/astro4dev/OAD-Data-Science-Toolkit" target="_blank">OAD-Data-Science-Toolkit</a> repository. Content is added by individual contributors by cloning the repository, making the changes, then creating a pull request. A description of the workflow can be found <a href="https://github.com/astro4dev/OAD-Data-Science-Toolkit/blob/master/How_to_upload_content.md" target="_blank">here</a>. Github members of the astro4dev data science working group are automatically informed about the new pull request. The request is then verified before it is added to the main astro4dev repository.
```

Comment:
Hi @veemcb  (can I still call you V ?)
I would suggest that each sentence is put on a new line, else git thinks the entire paragraph needs to be changed. This makes it very difficult to catch little errors and leads to churn in the code.